modules/olm-enabling-operator-for-multi-arch.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="olm-enabling-operator-for-multi-arch_{context}"]
+= Enabling your Operator for multiple architectures and operating systems
+
+As an Operator author, you can specify whether your Operator supports multiple
+architectures. If your Operator supports variants other than AMD64 and
+Linux, your CSV must list the supported variants for the Operator to run
+properly.
+
+.Supported architecture strings
+----
+386
+amd64
+amd64p32
+arm
+armbe
+arm64
+arm64be
+ppc64
+ppc64le
+mips
+mipsle
+mips64
+mips64le
+mips64p32
+mips64p32le
+ppc
+riscv
+riscv64
+s390
+s390x
+sparc
+sparc64
+wasm
+----
+
+.Supported operating system strings
+----
+aix
+android
+darwin
+dragonfly
+freebsd
+hurd
+illumos
+js
+linux
+nacl
+netbsd
+openbsd
+plan9
+solaris
+windows
+zos
+----
+
+[NOTE]
+====
+To support listing multiple architectures and operating systems, your Operator
```

Comment:
Do we want to note that it must be SHA digest reference just to align with the related images and disconnected stuff that will be below this?